### ISSUE TITLE
Adding support for players to use Nvidia Geforce Now cloud gaming service

### ISSUE THREAD

**Issue by alphachicken22** [tsunamayo/Starship-EVO#2189]
### Short Description of the issue:

...

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 
- Graphics Card: 

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
Okay I see - I wont have time for this on the short term but I will think about this! Thanks

**Comment by alphachicken22** [comment]
Thanks, all I wanted is to make you aware earlier than later. BTW Keep up the AMAZING work! I'm a Solo-Developer myself and I know what it's like.
...
I personally like Geforce Now because it unlocks some doors for people who don't have the best PCs for games, *Like me*, which also gives you a bigger player audience.

On the developer side this could be a great way to find bugs using the same hardware. Geforce Now host computers all use the same high quality hardware to my knowledge so it might be easier to correlate where problems arise.

**Comment by alphachicken22** [comment]
Is there any updates on getting Geforce NOW support for Starship EVO?

**Comment by tsunamayo** [comment]
Nope, I wont have time for this on the short - medium term, so dont hold your breath... Thanks for the cheers up!

**Comment by alphachicken22** [comment]
No problem, the life of a developer is difficult. Keep up the amazing work, we're all cheering for you!
